### PR TITLE
Remove `Backend` and use `tauri` `State`

### PR DESCRIPTION
**Pull request by etorreborre**
`Backend` was introduced in an attempt to be able to test the application without necessarily enrolling or resetting anything.
However, for now, it is a bit premature as we are still trying to find out what is the best application structure.

The other commit is using `tauri`'s managed `State` to instantiate the global options once and for all (as `quiet` so the terminal output is not mangled with regular logs) and pass it around.

### CODE REVIEW

**Review by shanesveller — APPROVED** [review]
This LGTM, no offense taken if you also wait for additional review from Davide/Adrian.

**Comment by etorreborre** [comment]
Thanks @shanesveller

**Pull request merged**
